src/navigation/ackermann_motion_primitives.cc

```diff
@@ -56,10 +57,22 @@ AckermannSampler::AckermannSampler() {
 
 void AckermannSampler::SetMaxPathLength(ConstantCurvatureArc* path_ptr) {
   ConstantCurvatureArc& path = *path_ptr;
+  const float stopping_dist =
+      Sq(vel.x()) / (2.0 * nav_params.linear_limits.max_deceleration);
+  if (!CONFIG_clip_cpoa) {
```

Comment:
This logic seems reversed - shouldn't this be `if (CONFIG_clip_cpoa)`? Investigate.